protopipe_grid_interface/scripts/submit_jobs.py

```diff
@@ -417,6 +417,34 @@ def main():
     log.debug("Particle type: %s", particle)
     log.debug("Energy estimation: %s", estimate_energy)
 
+    # Upload analysis configuration file for provenance
+    if switches["upload_analysis_cfg"]:
+        se_list = ["CC-IN2P3-USER", "DESY-ZN-USER", "CNAF-USER", "CEA-USER"]
```

Comment:
shouldn't this come from `grid.conf` rather than be hard-coded?

Reply:
I opened issue #58 about this (what is`grid.conf`?)

Reply:
I mean the list of sites is hard-coded here, but it could change.  Anyhow, could be a separate issue

Reply:
Sorry I meant #66

Reply:
though we could use that: we place this list there and the `analysis_metadata.yaml` already pinpoints its location in the analysis folder so it should work - I'll update the issue

Reply:
actually not, I'll open a new PR has it will change also at least another script